client/lib/cart-values/cart-items.js

```diff
@@ -662,6 +684,7 @@ module.exports = {
 	getDomainMappings,
 	getDomainRegistrations,
 	getDomainRegistrationTld,
+	getDependentProducts: getDependentProducts,
```

Comment:
there's no need to repeat ourselves here, especially since the other items don't